src/script/index.js

```diff
@@ -0,0 +1,21 @@
+const large = document.querySelector('.large');
+const bottomImg = document.querySelector('.bottom-img');
+
+bottomImg.onclick = function (event) {
+    let  target = event.target;
+    console.log(target);
+    while (target !== this) {
+        if (target.nodeName === 'A') {
+            return changeLargeImg(target.href, target.title);
+        }
+
+         target = target.parentNode;
+    }
+
+};
+
+function changeLargeImg(href, title) {
+    large.src = href;
+    large.alt = title;
+    return false;
```

Comment:
the function should do one action. This is a prevent default - second action. Do it in the parent function instead